lib/tmdb_handler.rb

```diff
@@ -1,94 +0,0 @@
-# frozen_string_literal: true
-
-module TmdbHandler
```

Comment:
Bye! Thanks for your service! #mariekondo

Reply:
So long TmdbHandler!